source/hassio/installation.markdown

```diff
@@ -28,7 +28,7 @@ The following will take you through the steps required to install Home Assistant
 2. Install Home Assistant OS:
 
    - **For the device images:** Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
-   - **For the virtual appliance images:** Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
+   - **For the virtual appliance images:** Load the appliance image into your virtual machine software. The appliance image initially has minimal free space and you should expand it to at least 32GB to avoid running out of space at runtime and during updates.  Home Assistant will automatically recognize and use the additional space the next time it starts.  An example command to resize the QCOW2 image is `qemu-img resize image.qcow2 +32G` (replace `image.qcow2` with the filename), and instructions for other image types are available via web search. You should assign RAM/CPU resources based on the guidance below and your anticipated add-on needs.
```

Comment:
I think this is too much for this space here.

PS: We don't use double spaces.